pkg/api/event.proto

```diff
@@ -132,6 +132,23 @@ message JobFailedEvent {
     Cause cause = 12;
 }
 
+message JobPreemptedEvent {
```

Comment:
You are having some python test failures because we generate some types from these event files.  You probably just need to add this event to the python tests.  

https://github.com/G-Research/armada/blob/master/client/python/tests/unit/test_gen.py

@Sharpz7 could help with this.